plugins/package-managers/go/src/main/kotlin/utils/GoSupport.kt

```diff
@@ -25,3 +25,24 @@ package org.ossreviewtoolkit.plugins.packagemanagers.go.utils
  * https://go.dev/ref/mod#incompatible-versions.
  */
 fun normalizeModuleVersion(moduleVersion: String): String = moduleVersion.removePrefix("v").substringBefore("+")
+
```

Comment:
This file only contains functions shared between `GoDep` and `GoMod`. For cohesion, can we move this into `GoMod` and reduce the visibility to `internal`?

Reply:
Agreed. Moved "escapeVersion" to GoMod.kt with `internal` visiblity.